### PR TITLE
Examples added for cache type

### PR DESCRIPTION
**Pull request by ajithkumar-maragathavel**
## Purpose of this pull request

This pull request (PR) adds the examples on the translate cache type implementation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/extension-dev-guide/cache/partial-caching/create-cache-type.html

### CODE REVIEW

File: src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md [hunk 1]
```diff
@@ -83,3 +83,51 @@ You must specify the following parameters:
...
+```xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Cache/etc/cache.xsd">
+    <type name="translate" translate="label,description" instance="Magento\Framework\App\Cache\Type\Translate">
+        <label>Translations</label>
+        <description>Translation files</description>
+    </type>
+</config>
+```
+
+Translate cache type model class is defined in `Magento\Framework\App\Cache\Type\Translate.php` class. It must extends `Magento\Framework\Cache\Frontend\Decorator\TagScope` class.
```
Comment (BarnyShergold) [inline src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md:100]:
```suggestion
Translate cache type model class is defined in `Magento\Framework\App\Cache\Type\Translate.php` class. It must extend the `Magento\Framework\Cache\Frontend\Decorator\TagScope` class.
```

File: src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md [hunk 2]
```diff
@@ -83,3 +83,51 @@ You must specify the following parameters:
...
+ */
+class Translate extends \Magento\Framework\Cache\Frontend\Decorator\TagScope
+{
+    /**
+     * Cache type code unique among all cache types
+     */
+    const TYPE_IDENTIFIER = 'translate';
+
+    /**
+     * Cache tag used to distinguish the cache type from all other cache
```
Comment (BarnyShergold) [inline src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md:121]:
```suggestion
     * Cache tag used to distinguish the cache type from all other caches
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just a couple of minor grammar corrections

File: src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md (hunk 1 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md]: Thanks for the review @BarnyShergold  I have committed the changes requested.

File: src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md (hunk 2 above)
Comment (ajithkumar-maragathavel) [reply src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md]: Fixed this..

**Review by BarnyShergold — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**